paasta_tools/utils.py

```diff
@@ -3268,8 +3268,8 @@ def set_paasta_print_file(file: Any) -> Iterator[None]:
 
 
 def paasta_print(*args: Any, **kwargs: Any) -> None:
-    f = kwargs.pop("file", sys.stdout)
-    f = getattr(TLS, "paasta_print_file", f)
+    f = kwargs.pop("file", sys.stdout) or sys.stdout
```

Comment:
why that `or ...` is necessary here and below?

Reply:
because the file kwarg here could be None, and when that is the case, this function stack traces because None doesn't make sense here.
See the stack trace here:
https://fluffy.yelpcorp.com/i/DZqpnPzklJjJPrrG5tMx15RfsQDvBvPQ.html